src/charm.py

```diff
@@ -255,10 +270,11 @@ def _mongodb_exporter_layer(self) -> Layer:
                     "override": "replace",
                     "summary": "mongodb_exporter",
                     # todo pass URI in correct way
-                    "command": f"mongodb_exporter --mongodb.uri={self.monitor_config.uri}  --collector.diagnosticdata --compatible-mode",
+                    "command": "mongodb_exporter --collector.diagnosticdata --compatible-mode",
```

Comment:
we should not pass the URI as a variable because then the password can be visible - this was a small improvement